### DISCUSSION TITLE
azd auth loginでエラー

### DISCUSSION THREAD

**Discussion by us2024-test** [Azure/azure-dev#3919] [Q&A]
こちらのものをデプロイして実装していましたが、
https://github.com/Azure-Samples/jp-azureopenai-samples/tree/main/5.internal-document-search

azd auth login
を実行したところ以下のエラーが生じました。
ERROR: logging in: failed to authenticate: server response error:
 Get "https://login.microsoftonline.com/organizations/v2.0/.well-known/openid-configuration": read tcp ●●●●●●:●●●●●●->●●●●●●●●●●●●: wsarecv: An existing connection was forcibly closed by the remote host.

ちなみに、
az loginは実装可能
azd auth login --debugの結果は以下の通りです。
PS C:\Users\●●●●\Documents\GitHub\jp-azureopenai-samples\5.internal-document-search> azd auth login --debug
2024/05/16 10:23:59 main.go:56: azd version: 1.9.2 (commit ●●●●●●●●●●●●●●●●●●●●)
2024/05/16 10:23:59 main.go:214: using cached latest version: 1.9.0 (expires on: 2024-05-16T06:27:26Z)
2024/05/16 10:23:59 project.go:120: Reading project from file 'C:\Users\●●●●\Documents\GitHub\jp-azureopenai-samples\5.internal-document-search\azure.yaml'
2024/05/16 10:23:59 middleware.go:108: running middleware 'debug'
2024/05/16 10:23:59 middleware.go:108: running middleware 'ux'
2024/05/16 10:23:59 middleware.go:108: running middleware 'experimentation'
2024/05/16 10:23:59 experimentation.go:42: assignment context: ●●●●●●●●;
2024/05/16 10:23:59 middleware.go:108: running middleware 'telemetry'
2024/05/16 10:23:59 telemetry.go:43: TraceID: ●●●●●●●●●●●●
2024/05/16 10:23:59 project.go:120: Reading project from file 'C:\Users\●●●●\Documents\GitHub\jp-azureopenai-samples\5.internal-document-search\azure.yaml'
ERROR: logging in: failed to authenticate: server response error:
 Get "https://login.microsoftonline.com/organizations/v2.0/.well-known/openid-configuration": read tcp ●●●●●●:●●●●●●->●●●●●●●●●●●●: wsarecv: An existing connection was forcibly closed by the remote host.

恐れ入りますがどなたかご教示いただけますと幸いです。

**Comment by rajeshkamal5050** [comment]
@us2024-test are you still running into this issue? cc @vhvb1989
